rbac/migration_tool/migrate.py

```diff
@@ -81,19 +81,30 @@ def migrate_role(
     The mappings are returned so that we can reconstitute the corresponding tuples for a given role.
     This is needed so we can remove those tuples when the role changes if needed.
     """
-    v2_role_bindings = v1_role_to_v2_bindings(role, default_workspace, current_bindings)
+    v2_role_bindings = v1_role_to_v2_bindings(role, str(default_workspace.uuid), current_bindings)
```

Comment:
Looks like this API here intends to take a Workspace?

```suggestion
    v2_role_bindings = v1_role_to_v2_bindings(role, default_workspace, current_bindings)
```

So maybe either that ^ or change the param to a UUID type and pass that

Reply:
Good catch, thanks. Updated!